federation/pkg/federation-controller/ingress/ingress_controller.go

```diff
@@ -53,6 +57,11 @@ const (
 	uidConfigMapName        = "ingress-uid"                                 // Name of the config-map and key the ingress controller stores its uid in.
 	uidConfigMapNamespace   = "kube-system"
 	uidKey                  = "uid"
+	ControllerName          = "ingress"
+)
+
+var (
+	RequiredResources = []unversioned.GroupVersionResource{extensionsapiv1beta1.SchemeGroupVersion.WithResource("ingresses"), apiv1.SchemeGroupVersion.WithResource("configmaps")}
```

Comment:
Ingress controller does not require config maps to be enabled in federation-apiserver (It requires them to be present in underlying kube-apiservers).
config map was added to federation-apiserver after federated ingress controller was written.